modules/gitops-release-notes-1-10-2.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="gitops-release-notes-1-10-2_{context}"]
+= Release notes for {gitops-title} 1.10.2
+
+{gitops-title} 1.10.2 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="fixed-issues-1-10-2_{context}"]
+== Fixed issues
+
+The following issue has been resolved in the current release:
+
+* Before this update, all versions of Argo CD `v2.8.3` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD to `v.2.8.9` and patching this vulnerability in the Argo CD API.
+[IMPORTANT]
+====
+Breaking change: The Argo CD API will no longer accept non-GET requests that do not specify application or JSON as their content type. Although the accepted content types list is configurable, do not disable the content type check completely.
+====
```

Comment:
Hi @Srivaralakshmi, in the preview page, the format is a little different from what https://github.com/openshift/openshift-docs/pull/70955 has for this section. Could you please take a look?